app/models/ForecastParameters.ts

```diff
@@ -9,11 +9,27 @@ export interface ForecastParameters {
 }
 
 export const DEFAULT_FORECAST_PARAMETERS: ForecastParameters = {
-  maintenanceCostPercentage: 0.0125, // percentage maintenance cost
+  maintenanceCostPercentage: 0.02, // percentage maintenance cost
   incomeGrowthPerYear: 0.04, // 4% income growth per year
   constructionPriceGrowthPerYear: 0.025, // 2.5%
   rentGrowthPerYear: 0.03, // 3%
   propertyPriceGrowthPerYear: 0.05, // 5%
   yearsForecast: 40, // 40 years
   affordabilityThresholdIncomePercentage: 0.35, // percentage of income to afford rent or purchase
 } as const;
+
+/**
+ * Converts maintenance spend string to number and creates forecast parameters
+ * @param maintenancePercentage - Maintenance spend value from form ("0.015" | "0.02" | "0.0375")
+ * @returns ForecastParameters with updated maintenance cost
+ */
+export function createForecastParameters(maintenancePercentage: string): ForecastParameters {
```

Comment:
I think there's probably scope to simplify a fair few things in this PR by making sure that `maintenancePercentage` (and `MAINTENANCE_LEVELS`) are numbers at all times. 

Even if it's a string value in the input, we can use Zod to refine it to a number for submission, and then keep it as a number going forward in out codebase.

Reply:
Ah great catch, I think I've just pushed a fix for this now.